src/main/java/com/exadel/discountwebapp/history/enums/OperationsEnum.java

```diff
@@ -0,0 +1,14 @@
+package com.exadel.discountwebapp.history.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum OperationsEnum {
```

Comment:
let's find more descriptive name. for e.g. OperationType

Reply:
renamed